### PR TITLE
Make executable

### PR DESCRIPTION
**Pull request by slwoodward**
Executable works! 
host, port and ruleset are now entered after starting program (either using py RunClient.py or clicking on executable's icon).

### CODE REVIEW

**Comment by slwoodward** [comment]
After merging consensus branch with Master I rebased this branch.  Recreated executable, and insured that it ran properly (with consensus at the end of the round).

File: source/client/UICardWrapper.py [hunk 1]
```diff
@@ -20,8 +20,14 @@ def getImage(card):
...
-        image_file = os.path.join('client', 'cardimages', 'card' + str(card.number) + suit_letter + '.png')
-        img = pygame.image.load(image_file)
+        # image_file = os.path.join('client', 'cardimages', 'card' + str(card.number) + suit_letter + '.png')
+        #debug: Back_Img = pygame.image.load(os.path.join('bundle_data', 'cardimages', 'cardBack.png'))
+        #
+        # image_file = os.path.join('bundle_data', 'cardimages', 'card' + str(card.number) + suit_letter + '.png')
+        # img = pygame.image.load(image_file)
+        image_index = str(card.number) + suit_letter
+        img = UIC.card_images[image_index]
+        # img = pygame.image.load(os.path.join('bundle_data', 'cardimages', image_file))
```
Comment (hwoodward) [inline source/client/UICardWrapper.py]: Eventually this will need to be cleaned up

Comment (slwoodward) [reply source/client/UICardWrapper.py]: Will do this and let you know.

**Comment by slwoodward** [comment]
I put instructions in the wiki.  I don't know why the spec file is not reaching the repository.

**Comment by hwoodward** [comment]
I can't find instructions in the wiki and the spec file is still missing. I can forgive the spec file, but the instructions need to be easy to find if someone were to fork the project. If they are in the wiki, add a link in the README file.

**Comment by slwoodward** [comment]
Instructions in Wiki moved to where they can easily be found.  Uploaded spec files to source folder.

**Pull request merged**